test/unit_tests/braket/circuits/test_free_parameter.py

```diff
@@ -0,0 +1,72 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import numpy as np
+import pytest
+
+from braket.circuits import FreeParameter
+
+valid_params = [
+    1,
+    np.pi,
+]
+
+
+@pytest.fixture
+def free_parameter():
+    return FreeParameter("theta")
+
+
+def test_is_free_param(free_parameter):
+    assert isinstance(free_parameter, FreeParameter)
+
+
+def test_equality():
```

Comment:
You might want to do this with fixtures.

Reply:
I think some of the same comments I had about testing Circuits apply here.

Reply:
Add in equality tests